CHANGELOG.md

```diff
@@ -33,6 +39,10 @@ and this project, at least loosely, adheres to [Semantic Versioning](https://sem
 - jump_flags_to_params now works if some JUMPs are present, never modifies the TOAs, and is idempotent
 - jump_params_to_flags is now idempotent and unconditionally sets the -jump flag to a correct state
 ### Changed
+- TOAs no longer need to be grouped by observatory
+- model.phase() now defaults to abs_phase=True when TZR* params are in the model
+- removed explicit download of IERS and leapsecond data (handled now by astropy)
+- The default version of TT(BIPM) uses BIPM2021
```

Comment:
This is weird, how did these entries wind up in this PR?

Reply:
In a previous version, there were two instances of CHANGED in the changelog.  I had tried to consolidate them when this PR was in progress prior to the last release.  Now it's gotten messier.  

I also think it would be good if part of the release procedure was to add the Unreleased piece back into the changelog.

Reply:
Indeed! Merges will still go wrong, though.